quickstart/config/kratos/email-password/identity.traits.schema.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
move to it's own subdirectory (`oidc-provider`) here https://github.com/ory/kratos/tree/master/contrib

Reply:
Removed, see ory/kratos#538